torch/_inductor/lowering.py

```diff
@@ -787,6 +787,15 @@ def bessel_j1(x):
     return make_pointwise(fn)(x)
 
 
+@register_lowering(
+    [aten.special_modified_bessel_i0],
+    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
+)
+def modified_bessel_i0(x):
+    fn = ops_wrapper("modified_bessel_i0")
+    return make_pointwise(fn)(x)
```

Comment:
BTW for simple pointwise lowerings like this you can use the `register_pointwise` helper
```suggestion
register_pointwise(
    aten.special_modified_bessel_i0,
    name="modified_bessel_i0",
    type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
)
```

And move the definition down by the other `register_pointwise` calls.

Reply:
Thanks for the hint! I ended up using
```python
register_pointwise_numeric(aten.special_modified_bessel_i0)
```
with prefix `special_` removed within `register_pointwise`.